src/EditorFeatures/CSharp/UseAutoProperty/UseAutoPropertyAnalyzer.cs

```diff
@@ -81,11 +83,16 @@ private bool CheckExpressionSyntactically(ExpressionSyntax expression)
         protected override ExpressionSyntax GetGetterExpression(IMethodSymbol getMethod, CancellationToken cancellationToken)
         {
             var getAccessor = getMethod.DeclaringSyntaxReferences[0].GetSyntax(cancellationToken) as AccessorDeclarationSyntax;
-            var firstStatement = getAccessor?.Body.Statements.SingleOrDefault();
```

Comment:
'SingleOrDefault()` throws if there are more than one item, so that can't be used here.